build.gradle

```diff
@@ -35,6 +35,6 @@ allprojects {
   }
 }
 
-task clean(type: Delete) {
+tasks.register('clean', Delete) {
```

Comment:
This is unrelated to this PR. Can you remove this?

Reply:
Sure.